internal/catalogmetadata/filter/bundle_predicates_test.go

```diff
@@ -62,6 +62,53 @@ func TestInMastermindsSemverRange(t *testing.T) {
 	assert.False(t, f(b3))
 }
 
+func TestHigherBundleVersion(t *testing.T) {
+	b1 := &catalogmetadata.Bundle{Bundle: declcfg.Bundle{
+		Properties: []property.Property{
+			{
+				Type:  property.TypePackage,
+				Value: json.RawMessage(`{"packageName": "package1", "version": "1.0.0"}`),
```

Comment:
Oh, I see. I guess we'd need some sort of factory is we want to validate on creation...